_pages/dev/ssr/server-side-rendering-error-handling.md

```diff
@@ -6,73 +6,83 @@ feature:
 
-Without SSR Error Handling, the following issues arise:
+If your Spartacus app was created before version 2211.29, to benefit from all aspects of SSR error handling, you must do the following:
 
-- with the default Angular SSR behavior (used under the hood of Spartacus SSR), when the asynchronous error occurs during the rendering (e.g. API call, runtime error in the code), rendering process will not stop and eventually returns possibly malformed HTML to the client with the improper HTTP status code `200`. This can lead to the client-side errors and the page not being rendered correctly.
-- when customer enters the URL unknown for the application, Spartacus returns an error page, but with misleading status `200`.
+- Enable the `propagateErrorsToServer` feature toggle to start propagating the errors caught during server-side rendering to the ExpressJS server, where eventually they will be properly handled. For more information, see [Propagating Errors To The Server](#propagating-errors-to-the-server).
+- Enable the `ssrStrictErrorHandlingForHttpAndNgrx` feature toggle to seal the Angular application from the asynchronous errors that occur in the NgRx flow and from HTTP calls during the rendering process. For more information, see [Strict Error Handling for HTTP And NgRx](#strict-error-handling-in-angular-for-http-and-ngrx).
+- Enable `ssrFeatureToggle.avoidCachingErrors` in `SsrOptimizationOptions` to not cache any pages where an error occurred during rendering. For more information, see [Cache management and error handling](#cache-management-and-error-handling).
+- Use the `defaultExpressErrorHandlers` middleware in `server.ts` to handle errors in ExpressJS. For more information, see [Using Default ExpressJS Error Handlers](#using-default-expressjs-error-handlers).
+- Ensure that the `provideServer()` config function is provided in the `app.server.module.ts` file. It contains elements that are required for SSR error handling to work properly.
 
-Both scenarios may drastically affect the SEO of the page due to fact the wrong status code in the response to the client might lead to indexing by Google the malformed pages and unknown URLs.
+**Note:** SSR error handling is introduced in Spartacus 2211.29. For new applications, the feature is enabled by default. For older applications, the feature is disabled by default, but you can enable it using the feature toggles mentioned just above. After a certain period, the feature toggles will be enabled by default in newer versions of Spartacus, but you still may need to perform some steps manually to fully enable the functionality of SSR error handling.
+
+**Note:** Together with SSR error handling, Spartacus provides `ssrFeatureToggles` that allow you to enable or disable specific features for the `OptimizedSsrEngine`. For more information, see [SSR Feature Toggles](TODO: add link to page when is ready).
```

Comment:
`major:` REMINDER: let's not forget to attach a link to a page about SSR feature toggles when ready.

Reply:
Definitely on my list of things to do -- and I will share the link as soon as it's ready, so you guys can review it 🙂